openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java

```diff
@@ -1011,4 +1015,24 @@ private static SemanticModel getSemanticModel(Path clientPath) throws ProjectExc
             throw new ProjectException(e.getMessage());
         }
     }
+
+    /**
+     * Prints a warning log, if passed numeric data format is not supported.
+     *
+     * @param typeDescriptorName name of the data type. ex: number, integer
+     * @param schema uses to generate the type descriptor name ex: int32, int64
+     * @return typeDescriptorName for invalid numeric data formats
+     */
+    public static String convertOpenAPINumericTypeToBallerina(final String typeDescriptorName,
+                                                                     final Schema<?> schema) {
+        try {
+            if (schema.getFormat() != null) {
+                return GeneratorUtils.convertOpenAPITypeToBallerina(schema.getFormat().trim());
+            }
+        } catch (BallerinaOpenApiException e) {
+            LOGGER.error(String.format("WARNING: the Ballerina schemas do not support with the `%s` format " +
+                    "for the `%s` data type", schema.getFormat(), schema.getType()));
```

Comment:
```suggestion
            LOGGER.error(String.format("WARNING: ballerina cannot support the` %s` format for the `%s` data type.", schema.getFormat(), schema.getType()));
```

Reply:
How about we print this to the outstream since it is the pactrice which is already following

Reply:
yes. +1 to print to outstream, to be consistent.

Reply:
@lnash94 @aneeshafedo How about the below message?
```
String.format("WARNING: schemas with type ` %s` and format `%s` are not supported in Ballerina client and service generation.", schema.getType(), schema.getFormat())
```

Reply:
Added an outstream log line